packages/manager/apps/telecom/src/components/telecom/telephony/telephony-voip-service.service.js

```diff
@@ -30,6 +33,19 @@ angular
               // how should we handle errors ?
               const telephonyGroup = new TelephonyGroup(item.value);
               groups[telephonyGroup.billingAccount] = telephonyGroup;
+
+              OvhApiTelephony.v6()
```

Comment:
```suggestion
              return OvhApiMe.v6().get()
                  .then((me) => OvhApiTelephony.v6()
```

Reply:
Hi @FredericEspiau, I'm not sure about this one, we are in a forEach loop, so unless OvhApiMe caches results, we're going to have lots of useless requests.
Maybe I should chain OvhApiMe result with the fetch of billing accounts, will try that.

Reply:
> Maybe I should chain OvhApiMe result with the fetch of billing accounts, will try that.

done